configure.ac

```diff
@@ -284,6 +284,7 @@ AH_TEMPLATE([LOCK_DIR], [Directory for lock file])
 AH_TEMPLATE([BUNDLES_DIR], [Directory to use for bundles])
 AH_TEMPLATE([CERT_PATH], [Location of update certificate])
 AH_TEMPLATE([MOTD_FILE], [motd file path])
+AH_TEMPLATE([MOM_DIR], [Directory to copy the MoM used for the completion code])
```

Comment:
If you set this to anything different from /var/tmp/swupd/ the bash script will fail. I would let this hardcoded on swupd because otherwise we would need to generate a bash script with that info.

Reply:
ok